app/src/components/Search/PokemonHit.js

```diff
@@ -56,150 +36,127 @@ function getRefinementName(value) {
+                          {getPropertyByPath(pokemon, refinement.attribute)}
+                        </strong>
+                      </Tag>
+                    </li>
+                  ))}
+                </ul>
               )}
-            </p>
-          </header>
-
-          <img src={spriteUrl} alt={pokemon.names.en} />
-
-          <p className="watermark-number">#{pokemon.id}</p>
-
-          <ul>
-            {pokemon.types.map(type => (
-              <li key={type.name}>
-                <TypeBadgeAlgolia type={type.name} />
-              </li>
-            ))}
-          </ul>
-
-          {statRefinements && (
-            <ul
-              style={{
-                flexWrap: "wrap",
-                lineHeight: 1.6
-              }}
-            >
-              {statRefinements.map(refinement => (
-                <li key={refinement.attribute}>
-                  <Tag>
-                    {getRefinementName(refinement.attribute.split(".")[1])}{" "}
-                    <strong>
-                      {getPropertyByPath(pokemon, refinement.attribute)}
-                    </strong>
-                  </Tag>
-                </li>
-              ))}
-            </ul>
-          )}
+            </div>
+          </div>
         </PokemonLink>
-
         <style jsx>{`
-          div {
-            display: block;
-            border-radius: 2px;
-            border: 1px solid;
-            border-left: 4px solid;
-            background: #fff;
-            box-sizing: border-box;
-            position: relative;
-          }
-
-          div > :global(a) {
-            padding: 20px;
-            text-decoration: none;
-            color: inherit;
-            display: grid;
-            grid-template-columns: 1fr 150px;
-            grid-template-rows: repeat(3, auto);
-            align-items: center;
-          }
-
-          div > :global(a:hover .watermark-number),
-          div > :global(a:focus .watermark-number) {
-            opacity: 0.7;
-          }
-
-          div > :global(a:hover img),
-          div > :global(a:focus img) {
-            transform: scale(1.25);
-          }
-
-          img {
-            position: relative;
-            z-index: 1;
-            height: 150px;
-            grid-row: span 3;
-            transition: transform 0.3s ease;
-          }
-
-          h3 {
-            font-weight: bold;
-            font-size: 2rem;
-            margin-bottom: 6px;
-          }
-
-          p {
-            font-size: 1.4rem;
-            opacity: 0.75;
-            line-height: 1.6;
-          }
-
-          strong {
-            font-weight: bold;
-          }
-
-          ul {
-            display: flex;
-          }
-
-          li:not(:last-of-type) {
-            margin-right: 4px;
+          .pokemon-name {
+            text-shadow: 0 0 30px var(--color-type-${pokemon.types[0].name.toLowerCase()}), 0 0 60px var(--color-type-${pokemon.types[0].name.toLowerCase()});
```

Comment:
For some reason prettier breaks this code, it formats `--color-type-${pokemon.types[0].name.toLowerCase()}` to `--color-type- ${pokemon.types[0].name.toLowerCase()}` (with a space before the backticks) and that breaks the generated CSS.

I played with it locally and a solution would be to extract `--color-type-${pokemon.types[0].name.toLowerCase()}` to a variable (which we could reuse in one of my comments above)

Reply:
Prettier is not automated yet but its on our roadmap